README.rst

```diff
@@ -16,12 +16,17 @@ MDAnalysis also includes widely used analysis algorithms in the `MDAnalysis.anal
 The MDAnalysis project uses an `open governance model`_ and is fiscally sponsored by `NumFOCUS`_. Consider making 
 a `tax-deductible donation`_ to help the project pay for developer time, professional services, travel, workshops, and a variety of other needs.
 
-.. image:: https://raw.githubusercontent.com/numfocus/templates/master/images/numfocus-logo.png
-  :height: 60px
-  :target: https://numfocus.org/project/mdanalysis
-  :align: center
-  :alt: NumFOCUS
-  
+.. raw:: html
+
+   <div align="center">
+      <a href="<PROJECT_NUMFOCUS_SPONSORSHIP_LINK>">
+         <img height="60px" 
+            src="https://raw.githubusercontent.com/numfocus/templates/master/images/numfocus-logo.png" 
+            align="center">
+      </a>
+   </div>
+
+
```

Comment:
(feel free to ignore) In the end result this feel a little bit crowded. At least style-wise, I'd vote for a newline between the logo and the next line?